hostinfo/hostinfo_linux.go

```diff
@@ -48,6 +49,15 @@ func linuxDeviceModel() string {
 	return ""
 }
 
+func getQnapQtsVersion(version_info []byte) string {
+	re := regexp.MustCompile(`QTSFW_([\d.]+)`)
+	match := re.FindSubmatch(version_info)
```

Comment:
FindStringSubmatch

Reply:
Made version_info be a string and switched to FindStringSubmatch

Reply:
Replaced use of regex